src/app/clusters/network-commissioning/app_config_dependent_sources.cmake

```diff
@@ -27,6 +27,7 @@ TARGET_SOURCES(
     "${CLUSTER_DIR}/constants.h"
     "${CLUSTER_DIR}/NetworkCommissioningCluster.cpp"
     "${CLUSTER_DIR}/NetworkCommissioningCluster.h"
+    "${CLUSTER_DIR}/NetworkCommissioningDriverDelegate.h"
```

Comment:
This does not seem app-config dependent.

Reply:
This is ok: cmake is unable to pull dependencies like cmake does, so we place all the files here (although .h technically does not matter). The logic should be:
  - GN has a hard split between app-dependent or not
  - Cmake includes everything